convert2rhel/subscription.py

```diff
@@ -973,8 +973,8 @@ def check_needed_repos_availability(repo_ids_needed):
         if repo_id not in avail_repos:
             # TODO: List the packages that would be left untouched
             loggerinst.warning(
-                "%s repository is not available - some packages"
-                " may not be replaced and thus not supported." % repo_id
+                "Some repositories are not available: %s. Some packages may not be replaced with"
```

Comment:
```suggestion
                "The '%s' repository is not available. Some packages may not be replaced with"
```
Since this is in a for loop, let's stick to singular.

Reply:
gotcha yeah I can definitely do that, Thank you for the help!

Reply:
Since it is in a loop, _and_ you're already keeping track of `all_repos_avail`, the "`Some packages may not be replaced...`" part could be moved to after the loop, and only output once. i.e:

```python
    all_repos_avail = True
    for repo_id in repo_ids_needed:
        if repo_id not in avail_repos:
            # TODO: List the packages that would be left untouched
            loggerinst.warning(
                "The '%s' repository is not available." % repo_id)
            utils.ask_to_continue()
            all_repos_avail = False
    if not all_repos_avail:
        loggerinst.warning(
            "Some packages may not be replaced with their"
            " corresponding RHEL packages when converting."
    else:
        loggerinst.info("Needed RHEL repositories are available.")
```

Or, if you want that shown before the `ask_to_continue()` (which makes sense), only add that part of the message on the first missing repo:
```python
    all_repos_avail = True
    missing_repo_msg = str(
        " Some packages may not be replaced with their"
        " corresponding RHEL packages when converting.")
    for repo_id in repo_ids_needed:
        if repo_id not in avail_repos:
            # Only add missing_repo_msg the first time
            extra_msg = missing_repo_msg if all_repos_avail else ""
            # TODO: List the packages that would be left untouched
            loggerinst.warning(
                "The '%s' repository is not available.%s"
                % (repo_id, extra_msg))
            utils.ask_to_continue()
            all_repos_avail = False
    if all_repos_avail:
        loggerinst.info("Needed RHEL repositories are available.")
```